### PR TITLE
Creating T-shirt.md

### PR DESCRIPTION
**Pull request by creativefisher**

### CODE REVIEW

File: _haikus/T-shirt.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: T-shirt!
+author: Raman
+---
+
+Haikus are great fun
```
Comment (ltagliaferri) [inline _haikus/T-shirt.md]: Add a `<br>` tag or two spaces `  ` at the end of this line to force a line break in markdown.

File: _haikus/T-shirt.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+So is haiku based hacking
```
Comment (ltagliaferri) [inline _haikus/T-shirt.md]: Add a `<br>` tag or two spaces `  ` at the end of this line to force a line break in markdown.

File: _haikus/T-shirt.md (hunk 1 above)
Comment (creativefisher) [reply _haikus/T-shirt.md]: done

File: _haikus/T-shirt.md (hunk 2 above)
Comment (creativefisher) [reply _haikus/T-shirt.md]: done

**Pull request merged**